packs/pathfinder-monster-core/fungus-leshy.json

```
@@ -251,7 +251,7 @@
                 "category": "defensive",
                 "deathNote": true,
                 "description": {
-                    "value": "<p>When a fungus leshy dies, a burst of primal energy explodes from its body, restoring [[/r 2d8[healing]]]{2d8 Hit Points} to each fungi creature in a @Template[emanation|distance:30]. This area is filled with fungi, becoming difficult terrain.</p>\n<p>If the terrain is not a viable environment for this fungi, they wither after 24 hours.</p>"
+                    "value": "<p>When a fungus leshy dies, a burst of primal energy explodes from its body, restoring @Damage[2d8[healing]]{2d8 Hit Points} to each fungi creature in a @Template[emanation|distance:30]. This area is filled with fungi, becoming difficult terrain.</p>\n<p>If the terrain is not a viable environment for this fungi, they wither after 24 hours.</p>"
```

```suggestion
                    "value": "<p>When a fungus leshy dies, a burst of primal energy explodes from its body, restoring @Damage[2d8[healing]] Hit Points to each fungi creature in a @Template[emanation|distance:30]. This area is filled with fungi, becoming difficult terrain.</p>\n<p>If the terrain is not a viable environment for this fungi, they wither after 24 hours.</p>"
```

The reason I want to remove these is in case anything changes the roll, like effects and whatnot. The label will auto update if we aren't specifying a label. Can't do that for splash just yet, but we should otherwise prefer the auto labels.